ios/xcode/MGLKit/MGLKView.mm

```diff
@@ -89,6 +89,14 @@ - (void)setContext:(MGLContext *)context
     _context = context;
 }
 
+- (void)setEnableSetNeedsDisplay:(BOOL)value
+{
+    if (value)
```

Comment:
> For MGLKView, the default is false because there is no custom `setNeedsDisplay` implementation. I will add a dummy method which throws an exception if you try to set it to true.

Actually `MGLKView`'s default `enableSetNeedsDisplay` is true. Since `setNeedsDisplay` is not override, the default implementation of the base class will trigger a redraw via `drawRect` method.

In this case, I think we can just log a warning message whenever the user calls this method with NO parameter.

Reply:
I see. I want the opposite behavior. The OS should not causing any drawing so I can control it myself.

Reply:
I have overridden `setNeedsDisplay` as discussed.